workflow/react-project/src/components/common/PasswordInput/index.jsx

```diff
@@ -0,0 +1,25 @@
+import { useState } from 'react';
+
+import { faEyeSlash } from '@fortawesome/free-regular-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+import './style.less';
+
+const PasswordInput = ({ title, errorsMessage, ...inputProps }) => {
+  const [isPasswordVisible, setPasswordVisible] = useState(false);
+
+  const onPasswordIconClick = () => setPasswordVisible(!isPasswordVisible);
```

Comment:
`onPasswordIconClick ` can be renamed to `toggleValueVisibility` (because not clear what is Password Icon)